docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 4
+sidebar_label: Momento Authentication
+title: Momento Authentication With Expiring Tokens
+description: Learn how to use expiring tokens to enhance the security of your application
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Momento authentication with expiring tokens
+
+To access Momento services from your application, a Momento auth token is required. The auth token is an alphanumeric string Momento generates that is unique to your account and the cloud provider's Region you select. When creating an auth token, you can set up one of two types of expiration:
```

Comment:
We've talked about calling this the Momento `endpoint` you select. Do we want to introduce that now? Because it's not completely accurate that it's just cloud provider's region.

Also, I don't think region should be capitalized.

Reply:
While I know there is more information in the JWT token, is that other info critical for users to know to be successful? If so, then let's get a list and add it to this doc.

Region is capitalized in AWS documents, blogs, etm., that's why I do it. I am open to changing it here and in other docs as well.